packages/ckeditor5-table/tests/commands/inserttablecommand.js

```diff
@@ -194,5 +194,156 @@ describe( 'InsertTableCommand', () => {
 				);
 			} );
 		} );
+
+		describe( 'auto headings', () => {
+			it( 'should have first row as a heading by default', async () => {
+				editor = await ModelTestEditor
```

Comment:
```suggestion
				const editor = await ModelTestEditor
```

Reply:
You should not override the editor from `beforeEach` because it won't be properly destroyed. Also this editor instance should be destroyed locally.